UM/Settings/ContainerQuery.py

```diff
@@ -87,9 +87,9 @@ def execute(self, candidates: Optional[List[Any]] = None) -> None:
         # cache if it's not there yet.
         key_so_far = (self._ignore_case, )  # type: Tuple[Any, ...]
         if candidates is None:
-            filtered_candidates = list(self._registry.metadata.values())
+            filtered_candidates: Union[ValuesView[Dict[str, Any]], List[Dict[str, Any]]] = self._registry.metadata.values()
```

Comment:
This _does_ mean that the type now changes during execution (well, technically it's always that Union... but _eh_). Not really 'clean', but it'll work.

( If this was C++ I'd sent it back I think :-p )

Reply:
Well, it quacks like a duck and waggles like a duck ;)